DependencyInjection/CmfSeoExtension.php

```diff
@@ -290,4 +296,29 @@ private function loadSitemapHandling($config, XmlFileLoader $loader, ContainerBu
             $container->removeDefinition($this->getAlias().'.sitemap.guesser.alternate_locales');
         }
     }
+
+    /**
+     * Configuration block to configure form.
+     *
+     * @param array            $config
+     * @param ContainerBuilder $container
+     */
+    private function loadFormConfiguration($config, ContainerBuilder $container)
+    {
+        $seoMetadataClass = null;
+        if (isset($config['data_class'])
+            && isset($config['data_class']['seo_metadata'])
+            && null !== $config['data_class']['seo_metadata']
```

Comment:
or just `if (!empty($config['data_class']['seo_metadata']))` . !empty is both isset and not null.